components/feature/search/src/main/assets/extensions/ads/manifest.template.json

```diff
@@ -0,0 +1,219 @@
+{
+  "manifest_version": 2,
+  "applications": {
+    "gecko": {
+      "id": "ads_telemetry@mozac.org"
```

Comment:
🔥 Let's not change the ID relative to https://github.com/mozilla-mobile/fenix/blob/master/app/src/main/assets/extensions/ads/manifest.template.json#L5.

With this we end up with two installed extensions and we would need migration code to uninstall the old one first!

Reply:
Let's just keep the ID the same and make sure we remove the extension from Fenix.

Reply:
Right! Thank you!

Reply:
Wanted to change the previous ids just because seeing in logcat / console that Mozilla loads an "ads" extension might seem a bit strange.

Reply:
Yeah, agreed the naming isn't great, but not worth the trouble right now.